src/Core/src/Handlers/Button/ButtonHandler.cs

```diff
@@ -15,7 +15,7 @@ public ButtonHandler() : base(ButtonMapper)
 
 		}
 
-		public ButtonHandler(PropertyMapper mapper) : base(mapper ?? ButtonMapper)
+		public ButtonHandler(PropertyMapper? mapper = null) : base(mapper ?? ButtonMapper)
```

Comment:
These null values are needed for my stupid simple injection. If there is no default, it will try and inject the `PropertyMapper`. We probably need to fix this, but somehow didn't cause issues with the MS DI. Might be because it used the other ctor?